### PR TITLE
Choose a WebChannel port

### PR DESCRIPTION
**Pull request by samtstern**
### Description

When starting the Firestore emulator, scan 10 ports for a place to serve WebChannel traffic.

This should probably not ship until the flag is renamed to `webchannel_port` which will make the code cleaner and life easier :surfing_man: :sunglasses: :sun_with_face: 

### Scenarios Tested

Tested with both available and unavailable ports.

### Sample Commands

N/A

### CODE REVIEW

**Comment by ryanpbrewster** [comment]
tempted to bail on this and put effort into serving webchannel traffic from the same port as gRPC + REST traffic

File: src/emulator/firestoreEmulator.ts [hunk 1]
```diff
@@ -54,7 +55,35 @@ export class FirestoreEmulator implements EmulatorInstance {
       });
     }
 
-    return javaEmulators.start(Emulators.FIRESTORE, this.args);
+    // Make a copy of the args
+    const commandLineArgs: any = Object.assign({}, this.args);
```
Comment (samtstern) [inline src/emulator/firestoreEmulator.ts]: Wont have to do this stuff once webchannel_port is implemented and then we can make things less stringly typed.

**Comment by samtstern** [comment]
@ryanpbrewster happy to bail on this if you think that is achievable!  But we need one or the other for the FireConf codelab.

**Comment by samtstern** [comment]
@ryanpbrewster now that #1584 is submitted this is simplified and ready for review.

**Pull request merged**